docs/tracing/docs/030-tracing-showcase.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Troubleshooting
+type: Troubleshooting
+---
+
+## Tracing Showcase
+
+### I have no micro-service or lambda configured to receive an event
+
+### Scenario
+
+There is an Event sent from the external system (for example: The Enterprise Commerce) but there is no lambda or microservice configured with the Event trigger.
+
+### Expected Traces
+
+Only the trace for the "publish" and initial services will be visible as shown belown
+
+![](assets/showcase-only-publish-overview.png)
+
+In the trace details, you can see the tags for the "publish-service"
+
+![](assets/showcase-only-publish-detail.png)
+
+---
+
+### I have a micro-service / lambda Event trigger configured
+
+In such case, you will also see the "webhook-service" in the trace along with the "publish-service". There will be more spans from Envoy. You can also see the spans from the lambda or service for a downstream call if the tracing context headers are propagated while making those calls.
```

Comment:
same case like in above comment, plus: what is the webhook-service. This is new naming that is not covered by even bus documentation at all. No background info.

Without further review I already suggest this docs are converted into a getting started guide, where you can first describe how to check first events are delivered and then after connecting lambda/service how the situation changes. Please make use of your local technical writer and take a look on current getting started guides and their structure to do the same.